pkg/chaosdaemon/ipset_server.go

```diff
@@ -49,41 +49,42 @@ func (s *DaemonServer) FlushIPSets(ctx context.Context, req *pb.IPSetsRequest) (
 	return &empty.Empty{}, nil
 }
 
-func flushIPSet(ctx context.Context, pid uint32, set *pb.IPSet) error {
+func flushIPSet(ctx context.Context, pid uint32, set *pb.IPSet, enterNS bool) error {
```

Comment:
I think the order of the parameters of all methods in this file should be reconstructed. Only when `enterNS` is true, `pid` is useful. So we'd better put `pid` and `enterNS` together. Furthermore, `enterNS` is required of `pid`, so `enterNS` should be ahead of `pid`. For example:

```golang
func flushIPSet(ctx context.Context, enterNS bool, pid uint32, set *pb.IPSet) error 
```